Informatics II/Lectures/Twihning/heapsort.c

```diff
@@ -0,0 +1,78 @@
+#include <stdio.h>
+#include <math.h>
+#define N 13
+
+int arr[N] = {0,2,7,8,3,9,4,1,6,11,14,12,17};
```

Comment:
one minor thing, could you please move this array initialisation line inside the main.